dsc/workflows/base/simple_csv.py

```diff
@@ -0,0 +1,67 @@
+import csv
+import logging
+from collections.abc import Iterator
+from typing import Any
+
+import smart_open
+
+from dsc.utilities.aws import S3Client
+from dsc.workflows.base import BaseWorkflow
+
+logger = logging.getLogger(__name__)
+
+
+class SimpleCSV(BaseWorkflow):
```

Comment:
Not required, but I think a docstring for this class could be helpful.  Maybe short-and-sweet for the time being, in case there are learnings for actual workflows that extend this, that would be helpful to work back into this base class.

But this feels like a pretty foundational class in this application, and even just _stubbing_ the docstring section feels like it could be helpful.

Reply:
Agreed!

Reply:
Short docstring added!